Source/Private/ASLayoutTransition.mm

```diff
@@ -254,7 +300,7 @@ - (ASSizeRange)transitionContext:(_ASTransitionContext *)context constrainedSize
   for (ASLayout *sublayout in layout.sublayouts) {
     if (idx > lastIndex) { break; }
     if (idx >= firstIndex && [indexes containsIndex:idx]) {
-      ASDisplayNode *node = (ASDisplayNode *)sublayout.layoutElement;
+      ASDisplayNode *node = ASDynamicCast(sublayout.layoutElement, ASDisplayNode);
```

Comment:
Is this change necessary? Not that this is a particularly hot path but `ASDynamicCast` incurs a runtime cost whereas a static cast is free (if you know it's safe.)

Reply:
Haha this has nothing to do with the rest of this does it. But I noticed the comment below which was was claiming a behavior that simple type casting doesn't actually have!

Reply:
// Ignore the odd case in which a non-node sublayout is accessed and the type cast fails
      if (node != nil) {

Reply:
I will move this into a separate #trivial PR as there are already plenty of moving parts here to keep straight ;)